app/src/main/res/values/strings.xml

```diff
@@ -11,7 +11,7 @@
     <string name="img_house_row_desc">Image for the given house name</string>
     <string name="home_logo">Home Logo</string>
     <string name="title_activity_settings">Settings</string>
-    <string name="no_household_info">No info yet!</string>
```

Comment:
I would keep this string and use it as a default message when the user doesn't have a household selected, for example when he isn´t register in any

Reply:
Since we're using a recycler view instead of a text view now, if there is no household selected then the members will just be empty

Reply:
I added it for good measure

Reply:
Perfect!